example/src/main/java/com/stripe/example/activity/CustomerSessionActivity.java

```diff
@@ -53,23 +66,8 @@ public void onStringResponse(String string) {
                         }
                     }));
 
+        CustomerSession.getInstance().retrieveCurrentCustomer(this);
```

Comment:
Technically this isn't part of the library -- should it have given any warning for? Or does it not alert because it's compiled in?  @RestrictTo(RestrictTo.Scope.LIBRARY)